--- a/src/geometry/manifold/Polygon2d-manifold.cc
+++ b/src/geometry/manifold/Polygon2d-manifold.cc
@@ -1,3 +1,4 @@
 #include "geometry/PolySet.h"
 #include "geometry/Polygon2d.h"
+#include "geometry/manifold/manifoldutils.h"
 #include "utils/printutils.h"
@@ -9,5 +10,5 @@
 std::unique_ptr<PolySet> Polygon2d::tessellate() const
 {
   PRINTDB("Polygon2d::tessellate(): %d outlines", this->outlines().size());
-  return std::make_unique<PolySet>(*this);
+  return ManifoldUtils::createTriangulatedPolySetFromPolygon2d(*this);
 }
